### PR TITLE
build: add missing dependencies: pica, lodash

### PR DESCRIPTION
**Pull request by 7flash**
before installed them, following errors were appearing:

```
Failed to resolve import "pica" from "../../packages/excalidraw/src/data/blob.ts"
Failed to resolve import "lodash" from "../../packages/excalidraw/src/components/LibraryMenuItems.tsx".
```

### CODE REVIEW

**Review by ad1992 — CHANGES_REQUESTED** [review]
Hey @7flash good catch!
We should definitely add `pica ` as a dependency. As per lodash, lets add the ones used instead of adding the whole library, so `lodash.chunk` is what needs to be added and wherever we are using `throttle` lets import from `lodash.throttle`

**Comment by dwelle** [comment]
The `pica` comes with `pako` so it's strange it complains for you — but yes, we can make it an explicit. 

As for `lodash`, we're mistakenly importing the `chunk` from lodash instead from our utils. Fixed.

**Comment by ad1992** [comment]
> The pica comes with pako so it's strange it complains for you — but yes, we can make it an explicit.

Even though it comes as part of some other library but still since we are using it we should have it in package.json dependencies

**Review by dwelle — APPROVED** [review]

**Comment by ad1992** [comment]
I think there is still one lodash missed out in https://github.com/excalidraw/excalidraw/blob/master/src/excalidraw-app/collab/Portal.tsx#L17, we should use `lodash.throttle` instead

File: package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "open-color": "1.9.1",
     "pako": "1.0.11",
     "perfect-freehand": "1.0.16",
+    "pica": "^7.1.1",
```
Comment (ad1992) [inline package.json:45]: @7flash also can you lock the version ? as we are doing that for all deps so it has same build (lock file), you will need to run run `yarn` again to update lock file

**Review by ad1992 — APPROVED** [review]
looks good, merging
thanks @7flash!

**Pull request merged**